mycroft/skills/settings.py

```diff
@@ -216,6 +218,9 @@ def upload(self):
         The settingsmeta file does not change often, if at all.  Only perform
         the upload if a change in the file is detected.
         """
+        if not self.sync_enabled:
+            LOG.debug("Skill settings sync is disabled, skipping upload")
```

Comment:
This log message should be emitted once in the constructor as a INFO level message.  Putting it here will unnecessarily pollute the logs when DEBUG is turned on.

Reply:
i can change that, but it will hide how often mycroft is phoning home, which was my rationale behind DEBUG and not INFO, since it wont show by default 

if i am in debug mode i would need to scroll all way to the top to see this and most likely miss it. In general logs should happen where the action happens, they are logs not promises for the user to keep track off and hunt down the relevant strings... we will loose the distinction of upload vs download also

Essentially i am logging "mycroft tried to upload/download settings and was blocked" and you are telling me to change it to "this settings is disabled", there is a slight semantic change


i don't mind changing this, just explaining my rationale, mostly irrelevant in this case unless you are debugging functionality related to this (why arent my settings syncing kind of debugging)

Reply:
Perhaps as a compromise, an info message in the constructor for those that do not have debug turned on and leaving this here for those that do?

Reply:
that also works, but if you do feel the logs get very polluted i can remove it, i dont have a strong opinion about this. Haven't really investigated how often this will be printed

Reply:
This will run on a one mintue timer for each installed skill, so yeah it will pollute the logs if you have debug turned on.